tests/xtro-sharpie/MacCatalyst-SafariServices.ignore

```diff
@@ -0,0 +1,2 @@
+## for compatibility we have extra code that return `false` if `url` is `null`
+!extra-null-allowed! 'System.Boolean SafariServices.SSReadingList::SupportsUrl(Foundation.NSUrl)' has a extraneous [NullAllowed] on parameter #0
```

Comment:
This is a copy&paste from the iOS ignore, since it issuing to happen in all platforms (they all share code) it should be moved to the common file rather than having the same string in both files (and same explanation).

From the iOS todo:
```
iOS-SafariServices.ignore-## for compatibility we have extra code that return `false` if `url` is `null`
iOS-SafariServices.ignore:!extra-null-allowed! 'System.Boolean SafariServices.SSReadingList::SupportsUrl(Foundation.NSUrl)' has a extraneous [NullAllowed] on parameter #0
```